### PR TITLE
Add Vector Collection CR doc [CN-1361]

### PR DESCRIPTION
**Pull request by SeriyBg**

### CODE REVIEW

File: docs/modules/ROOT/pages/vector-collection-configuration.adoc [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+|Field|Description
+
+|name
+|Name of the vector collection configuration that you're creating. If empty, the custom resource (CR) name is used.
+
+|hazelcastResourceName
+|Defines the name of the Hazelcast resource for which the vector collection configuration is created.
+
+|indexes
+|Information about indexes configuration.
```
Comment (kutluhanmetin) [inline docs/modules/ROOT/pages/vector-collection-configuration.adoc:21]: Maybe we can add that at least one index is required.

**Review by kutluhanmetin — APPROVED** [review]
just a tiny comment, looks good!

File: docs/modules/ROOT/pages/vector-collection-configuration.adoc [hunk 2]
```diff
@@ -0,0 +1,45 @@
+= Configuring Vector Collection
```
Comment (oliverhowell) [inline docs/modules/ROOT/pages/vector-collection-configuration.adoc:1]:
```suggestion
= Configuring Vector Collections
```

File: docs/modules/ROOT/pages/vector-collection-configuration.adoc (hunk 1 above)
Comment (oliverhowell) [inline docs/modules/ROOT/pages/vector-collection-configuration.adoc:21]:
```suggestion
|Information about configuration for indexes.
```

File: docs/modules/ROOT/pages/vector-collection-configuration.adoc [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+include::ROOT:example$/vector-collection.yaml[]
+----
+
+After creating the `VectorCollection` resource, use the following command to check its status:
+
+[source,shell]
+----
+kubectl get vectorcollection vectorcollection-sample -o wide
+----
+
```
Comment (oliverhowell) [inline docs/modules/ROOT/pages/vector-collection-configuration.adoc:41]:
```suggestion
 
This should produce output like this:
```

**Review by oliverhowell — APPROVED** [review]
Minor suggestions but approving in advance

**Pull request merged**